pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_user-roles-in-pantheon.adoc

```diff
@@ -0,0 +1,20 @@
+[id="user-roles-in-pantheon_{context}"]
+= User roles in {ProductShortName}
+
+In {ProductShortName}, each user is assigned a role that determines the actions the user can perform.
+
+There are two user roles defined in {ProductShortName} - `Author` and `Publisher`. The following permissions are associated with the roles:
+
+`Author`::
+
+* Use Git to import their modules files into {ProductShortName}.
+* Revise existing modules.
```

Comment:
Replace `modules` with `content` everywhere, unless the action really does apply just to modules.